<launch>
  
  <!-- Write point clouds to file -->
  <node pkg="point_cloud_io" type="write" name="write" output="screen">
    <param name="topic"                  value="/camera/depth/points"/>
    <param name="folder_path"            value="$/home/magnuss/Documents/pointclouds"/>
    <param name="file_prefix"            value="received_cloud" />
    <param name="file_ending"            value="ply" />
    <param name="add_counter_to_path"    value="true" />
    <param name="add_frame_id_to_path"   value="false" />
    <param name="add_stamp_sec_to_path"  value="false" />
    <param name="add_stamp_nsec_to_path" value="false" />
  </node>

</launch>
